frontend_tests/node_tests/stream_data.js

```diff
@@ -491,3 +491,28 @@ zrequire('stream_data');
     page_params.notifications_stream = 'jazy';
     assert(!stream_data.notifications_in_home_view());
 }());
+
+(function test_create_sub() {
+    var india = {
+        stream_id: 102,
+        name: 'India',
+        subscribed: true,
+    };
+    var canada = {
+        name: 'Canada',
+        subscribed: true,
+    };
+    stream_data.create_sub_from_server_data('India', india);
+    stream_data.clear_subscriptions();
```

Comment:
Not sure what is the point of doing `stream_data.clear_subscriptions` and then creating another sub?